app/assets/stylesheets/modules/top_bar.scss

```diff
@@ -0,0 +1,44 @@
+@import 'variables';
+
+.top-bar {
+  background: $topbar-bg-color;
+
+  .row {
+    background-color: $topbar-bg-color;
+  }
+
+  .name h1 {
+    color: #FFFFFF;
+    width: 75%;
+    display: block;
+    padding: 0 0.9375rem;
+    font-family: Lobster;
+    width: 300px;
+  }
+
+  .row {
+    width: 100%;
```

Comment:
Properties should be ordered margin-bottom, margin-left, margin-right, margin-top, max-width, width